addon/components/search-input.js

```diff
@@ -1,72 +1,40 @@
 /* eslint-disable ember/no-classic-components, ember/no-classic-classes, ember/require-tagless-components, prettier/prettier, ember/no-get, ember/no-actions-hash */
 import { getOwner } from '@ember/application';
 import Component from '@ember/component';
-import { get, set } from '@ember/object';
-import { and } from '@ember/object/computed';
-import { inject as service } from '@ember/service';
-import { task, timeout } from 'ember-concurrency';
 
 import layout from '../templates/components/search-input';
 
-const SEARCH_DEBOUNCE_PERIOD = 300;
-const SEARCH_CLOSE_PERIOD = 200;
+import docsearch from '@docsearch/js';
+
+import '@docsearch/css';
 
 export default Component.extend({
   layout,
 
   classNames: ['search-input'],
 
-  searchService: service('search'),
```

Comment:
This service can likely also be removed, but unclear if that would be a breaking change for consumers of this package.